docs/introduction.md

```diff
@@ -0,0 +1,155 @@
+make it production ready. What if you could peel back the cover in a sense and tweak the packaged files to
+satisfy your needs?  Ansible Playbook Bundle is designed to make this transition from simple to complex workflows
+possible.
+
+To introduce the Ansible Playbook Bundle concept we will cover:
+
+  1.) Service Catalog and Ansible Service Broker workflow
+
+  2.) Ansible Playbook Bundle overview
+
+  3.) Ansible Playbook Bundle workflow
+
+    a.) Preparing an Ansible Playbook Bundle
+
+    b.) Packaging an Ansible Playbook Bundle
+
+    c.) Deploying an Ansible Playbook Bundle
+
+### Ansible Service Broker
+
+In December 2016 CloudFoundry [announced](https://www.cloudfoundry.org/open-service-broker-api-launches-as-industry-standard/) the open sourcing of its service broker APIs, [Open Service Broker API](https://www.openservicebrokerapi.org/).
+The Open Service Broker API defines methods for creating a broker, an entity responsible for delivering
+applications or services to a cloud platform. The Ansible Service Broker is a specialized broker created
+for OpenShift to manage Ansible Playbook Bundles.
+
+Users will interact with the [Service Catalog](https://github.com/kubernetes-incubator/service-catalog) to obtain a list of available applications and invoke operations
+for provisioning, deprovisioning, binding, and unbinding to an application. The Service Catalog will rely on
+a collection of brokers to handle details associated with its applications. The diagram below illustrates a
+high level concept of the workflow.
+
+![Overview](ansible-service-broker-overview.png)
+
+**Service Catalog to Service Broker Workflow**
+
+  1.) User requests list of available applications from the Service Catalog
+
+  2.) Service Catalog asks the Ansible Service Broker for available applications
+
+  3.) Ansible Service Broker talks to a container registry to learn which APBs are available
+
+  4.) User issues a command to provision a specific APB
+
+  5.) Provision request makes its way to the Ansible Service Broker which fulfills the command by invoking the
+      provision method on the APB
+
+### Ansible Playbook Bundle: Overview
+
+An Ansible Playbook Bundle (APB) borrows several concepts from the [Nulecule](https://github.com/projectatomic/nulecule) or [Atomicapp](http://www.projectatomic.io/docs/atomicapp/) project, namely the concept of a short
+lived container with the sole purpose of orchestrating the deployment of the intended application. For the case
+of APB, this short lived container is the APB; a container with an Ansible runtime environment
+plus any files required to assist in orchestration such as playbooks, roles, and extra dependencies.
+Specification of an APB is intended to be lightweight, consisting of several named playbooks and a
+metadata file to capture information such as parameters to pass into the application.
+
+The workflow for an APB is broken up into three steps:
+
+  1.) Prepare
+
+  2.) Package
+
+  3.) Deploy
```

Comment:
Same comment about above numbered list. Probably applies to any other ones in the document.